### ISSUE TITLE
Project Feedback

### ISSUE THREAD

**Issue by mdennis1318** [mdennis1318/MI-449-SS17-740-js-intro-Vh2K33#1]
@KatieMFritz Can you take a look at this? It's [hosted here](https://mdennis1318.github.io/MI-449-SS17-740-js-intro-Vh2K33/) and meets the following criteria:

- [x] At least 3 `button` elements are included in your page
- [x] When each of the buttons are hovered over (`mouseenter` event) or clicked on (`click` event), a unique sound effect plays from an `audio` element (try the `.play()` method) -- you may use [these drum kit sounds](/static/audio/drum-sounds.zip), create ones from [this sound effects generator](http://sfbgames.com/chiptone/), or use any other sounds that won't hurt/irritate the people who listen to them
- [x] Each button is well-labeled, corresponding to the sound it will play
- [x] All `audio` elements are invisible (no `controls` attribute), so that they can only be played with the buttons
- [x] All of your JavaScript is in a separate file ending with `.js`

**Comment by KatieMFritz** [comment]
Wow, nice touch with the image! 🥁 🎵 

You've also done a fantastic job setting up your project in an easy-to-understand way, including:

- logical, consistent, descriptive element ids and variable names that follow punctuation and capitalization conventions ✨ 
- putting your sound files into a separate folder to get them out of the way 👍 
- clear, descriptive commit messages 💥 

Here are a few really small improvements to make (in the interest of helping you level up):

### Getting rid of `.DS_Store` clutter

OS X automatically generates hidden `.DS_Store` files and by default, Git will include them your commits. To get rid of this clutter in our Git repos, you can [check out this link](http://stackoverflow.com/questions/107701/how-can-i-remove-ds-store-files-from-a-git-repository).

### Extra line break

There's an extra line break at line 22 of your `index.html`.

### Make use of Bootstrap

_You don't have to actually change this if you don't feel like it, but I wanted you to know._

I notice you're linking to Bootstrap CSS, but you're not really using it for anything Bootstrappy. If you just want to make your page load more consistently across browsers, try [normalize.css](https://cdnjs.com/libraries/normalize) - it's actually part of Bootstrap!

If you _do_ want to make more use of Bootstrap, some easy things you could do are:

- give your buttons the `btn` or `btn btn-primary` class, or even make them into a [button group](http://getbootstrap.com/components/#btn-groups)
- wrap your content in `<div class="container">` to center it
- put the content in `<div class="jumbotron">` to give it some special formatting (play around with nesting `container` and `jumbotron`)

After you’ve made those two small changes (and css changes, if you want) and pushed them to GitHub and your hosted site, give it a once-over to make sure it looks right, then comment back here and I’ll take another look.

Thanks! :rocket:

**Comment by mdennis1318** [comment]
@KatieMFritz I have updated the git repo to remove .DS_Store along with removing the extra line break at line 22. I have also change the stylesheet to normalize.css. Therefore, I think it is ready for another look!

Thanks!

**Comment by KatieMFritz** [comment]
Hi Matt, good job with the .DS_Store stuff and switching to normalize.css! 👍 

Still seeing an extra line break, though, this time at [line 25 of index.html](https://github.com/mdennis1318/MI-449-SS17-740-js-intro-Vh2K33/blob/master/index.html#L25)
```html
      <audio id="bass-sound" src="drum-sounds/kick.wav"></audio>
      <audio id="snare-sound" src="drum-sounds/snare.wav"></audio>


    <script src="script.js"></script>
```

And now the elements inside `body` (except for the `script` at the end) are indented an extra level. Let me know if you're seeing something different or have questions.

This is super close - just want to get it 100% there before I approve. 😄

**Comment by mdennis1318** [comment]
@KatieMFritz Yeah that was careless of me! Sorry I fixed those two things. One more look please!

Thanks!

**Comment by KatieMFritz** [comment]
There we go. Looks wonderful! 😁 

:shipit:
